app/src/main/java/com/example/fittracker/mvp/view/AvatarListView.java

```diff
@@ -0,0 +1,46 @@
+package com.example.fittracker.mvp.view;
+
+import android.app.Activity;
+import android.support.v7.widget.LinearLayoutManager;
+import android.support.v7.widget.RecyclerView;
+import android.view.View;
+import android.widget.ProgressBar;
+
+import com.example.fittracker.R;
+import com.example.fittracker.adapter.MarvelAdapter;
+import com.example.fittracker.mvp.contract.AvatarListContract;
+import com.example.fittracker.services.marvel.Result;
+
+import java.lang.ref.WeakReference;
+import java.util.ArrayList;
+
+import butterknife.BindView;
+import butterknife.ButterKnife;
+
+public class AvatarListView implements AvatarListContract.View {
+    private WeakReference<Activity> activity;
+    @BindView(R.id.avatarlist_recyclerview)
+    RecyclerView recyclerView;
+    @BindView(R.id.avatarlist_progress_bar)
+    ProgressBar progressBar;
+
+    public AvatarListView(Activity activity) {
+        this.activity = new WeakReference<>(activity);
+        ButterKnife.bind(this, activity);
+        recyclerView.setAdapter(new MarvelAdapter(new ArrayList<Result>()));
+        LinearLayoutManager llm = new LinearLayoutManager(this.activity.get());
+        recyclerView.setLayoutManager(llm);
```

Comment:
put this last three lines in a separate method. Something like initAdapter

Reply:
Done